pkg/storage/inmemstore/store.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package inmemstore
```

Comment:
@acud Maybe we should combine this and the mock store. Both are essentially the same.